frontend/server/src/ProblemParams.php

```diff
@@ -39,6 +39,11 @@ class ProblemParams {
     // Custom validator
     const VALIDATOR_CUSTOM = 'custom';
 
+    // Diffs to show when problem is educational
+    const NO_SHOW_DIFFS = 'none';
+    const SHOW_ONLY_EXAMPLE_DIFF = 'examples';
+    const SHOW_ALL_DIFFS = 'all';
```

Comment:
aquí la idea es que si hay cosas semánticamente relacionadas, deberían tener el mismo prefijo (checa todos los `VALIDATOR_*` y `UPDATE_PUBLISHED_*`, `VISIBILITY_*` arribita). entonces qué te parece usar `SHOW_DIFFS_` como prefijo?

```suggestion
    const SHOW_DIFFS_NONE = 'none';
    const SHOW_DIFFS_EXAMPLES = 'examples';
    const SHOW_DIFFS_ALL = 'all';
```

Reply:
Listo